docs/legacy/metricset-indices.asciidoc

```diff
@@ -1,9 +1,5 @@
 [[metricset-indices]]
-== Metrics documents
-
-++++
-<titleabbrev>Metrics documents</titleabbrev>
-++++
+== Application metrics
```

Comment:
IMO we'll need to rework the metrics docs for some reasons:
- the main question that we see raised is which metrics end up in service specific datastreams and which in the generic one
- irritation around the aggregated metrics

We'll need to better cover these topics, but without over-documenting it, as the concrete (aggregated) metrics data streams are considered an internal detail, that we might change, as long as the UI changes the queries in a compatible way. 
Should we have an in-depth conversation for this topic on this PR, or do you prefer a follow up issue to not slow down the structural changes?

Reply:
👍 Makes sense. I'd prefer to move the conversation to a separate issue.